lively.collab/comments/commentMorph.js

```diff
@@ -70,31 +69,38 @@ export class CommentGroupMorph extends Morph {
   }
 
   getCommentMorphCount () {
-    return this.ui.commentMorphContainer.submorphs.length;
+    return this.commentMorphs.length;
   }
 
   getUnresolvedCommentMorphCount () {
-    return this.ui.commentMorphContainer.submorphs.filter((commentMorph) => !commentMorph.comment.isResolved()).length;
+    return this.commentMorphs.filter((commentMorph) => !commentMorph.comment.isResolved()).length;
   }
 
   async removeCommentMorph (comment) {
-    this.ui.commentMorphContainer.submorphs.forEach((commentMorph) => {
+    this.commentMorphs.forEach((commentMorph) => {
       if (commentMorph.comment.equals(comment)) {
-        commentMorph.remove();
+        commentMorph.delete();
+        remove(this.commentMorphs, commentMorph);
+        this.updateCommentContainerSubmorphs();
```

Comment:
Might it be possible and more efficient to put it below above `this.updateCommentCountLabel`?

Reply:
Why do you think this would be more efficient?

Actually we would have put the updateCommentCountLabel inside the conditional as well.